doc/user_manual/sampler.tex

```diff
@@ -2718,6 +2743,7 @@ \subsubsection{Adaptive Sobol Decomposition}
       \item \distributionDescription
     \item \functionDescription
     \end{itemize}
+    \item \constantVariablesDescription
```

Comment:
Maybe not worth fixing now, but we should look at encapsulating the documentation that is common to each sampler so future additions are less repetitious.